packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -610,19 +637,21 @@ export const components: ThemeOptions["components"] = {
         ...(ownerState.endAdornment && {
           paddingRight: theme.spacing(3),
         }),
+        ...(ownerState.multiline && {
+          padding: "0",
+          ...(ownerState.size === "small" && {
+            padding: "0",
+          }),
+        }),
       }),
       input: ({ theme }) => ({
         padding: `calc(${theme.spacing(3)} - 1px) ${theme.spacing(3)}`,
-        border: "1px solid transparent",
+        borderWidth: theme.mixins.borderWidth,
+        borderStyle: theme.mixins.borderStyle,
+        borderColor: "transparent",
```

Comment:
I like that we separate these out. I also do it for padding to make my intent more clear.